ext/mbstring/mbstring.c

```diff
@@ -4275,13 +4275,6 @@ MBSTRING_API char *php_mb_safe_strrchr_ex(const char *s, unsigned int c, size_t
 }
 /* }}} */
 
-/* {{{ MBSTRING_API char *php_mb_safe_strrchr() */
-MBSTRING_API char *php_mb_safe_strrchr(const char *s, unsigned int c, size_t nbytes)
```

Comment:
I'd suggest to also rename php_mb_safe_strrchr_ex to php_mb_safe_strrchr.

Reply:
Will do.